drivers/flash/flash_stm32_ospi.c

```diff
@@ -226,6 +229,18 @@ static int ospi_read_access(const struct device *dev, OSPI_RegularCmdTypeDef *cm
 	struct flash_stm32_ospi_data *dev_data = dev->data;
 	HAL_StatusTypeDef hal_ret;
 
+#if CONFIG_FLASH_STM32U5_ES0553_2_5_13
+	int err;
+
+	if (dev_data->errata_u59_2_5_13) {
+		dev_data->errata_u59_2_5_13 = false;
+		err = ospi_read_access(dev, cmd, data, 1);
```

Comment:
@FRASTM can you confirm this is fine ?

Reply:
according to the errata sheet, it is only needed after exiting stop 2 or 3. but then we need a way to get notified about that, which AFAIK may be possible with PM hooks. should we go that way?